proto/backend.proto

```diff
@@ -21,13 +21,56 @@ message PluginConfig {
 //-----------------------------------------------
 
 service Core {
+  // Information about what a backend plugin supports/expects
+  rpc GetSchema(GetSchema.Request) returns (GetSchema.Response);
```

Comment:
Why does grafana core need this?  Can we delegate that to the plugin?

Rather than `GetSchema` could we have:
```
rpc GetOpenAPI(request) returns (response);
```
the response would be identical to the CallResource request... or empty.

alternatively (maybe better?) we could put it on the `/api` resource path?

Reply:
Same reason as https://github.com/grafana/grafana-plugin-sdk-go/pull/48#discussion_r370223251

But from an SDK standpoint, we can provide one simplified API if you only want one resource catch all handler and a more advanced API where you can define resources/routes.

OpenAPI spec doesn't feel like the right fit for this use case, but I may be wrong. Lets discuss this later today.